Foundation/NSFileHandle.swift

```diff
@@ -214,11 +214,15 @@ extension FileHandle {
     open class var standardError: FileHandle {
         return _stderrFileHandle
     }
-    
+
+    internal static var _nulldeviceFileHandle: FileHandle = {
+        return FileHandle(forUpdatingAtPath: "/dev/null")!
```

Comment:
What's the portability story around `/dev/null`? 

For what it's worth, the Darwin Foundation implementation here is a simple subclass of FileHandle that returns empty `Data` and values of `0` (except for file handle, which is `-1`).